tools/ovc/openvino/tools/ovc/cli_parser.py

```diff
@@ -871,7 +871,22 @@ def get_model_name_from_args(argv: argparse.Namespace):
     if not isinstance(input_model, (str, pathlib.Path)):
         return output_dir
 
-    input_model_name = os.path.splitext(os.path.split(input_model)[1])[0]
+    # get file name
+    input_model_name = os.path.split(input_model)[1]
+
+    # get base directory name if file name is empty
+    while input_model_name == '' and len(input_model) > 0:
+        input_model_name = os.path.basename(input_model)
+        input_model = os.path.split(input_model)[0]
+
+    # remove extension if exists
+    input_model_name = os.path.splitext(input_model_name)[0]
```

Comment:
pay attention to that `os.path.splittext` can raise exception of `input_model_name` is still ''

Reply:
`os.path.splittext()` always returns tuple with two  elements. I checked the result of `os.path.splitext('')`
It returns:` ('', '')`
So this line looks safe.